qualang_tools/bakery/bakery.py

```diff
@@ -9,20 +9,50 @@
 import copy
 
 
-def baking(config, padding_method="right"):
-    return Baking(config, padding_method)
+def baking(
+    config,
+    padding_method="right",
+    override=False,
+    update_config: bool = True,
+    baking_index: int = None,
+):
+    """
+    Opens a context manager to synthesize samples for arbitrary waveforms
+    :param config: config file
+    :param padding_method: Method to pad 0s to format the waveform to match hardware constraint
+    (>16 ns and multiple of 4)
+    :param override: Define if baked waveforms are overridable when using add_compiled feature
+    :param update_config: Define if baked waveform should be added within the input config file
+    :param baking_index: index of a reference baking object to impose length constraint on new baked waveform
+    (useful for matching lengths when using waveform overriding in add_compiled feature)
+    :return: No return, config is updated if update_config is set to True,
```

Comment:
If there is no return, don't add a :return: part...
You can write that part in the docstring itself above